modules/twinkleprotect.js

```diff
@@ -517,6 +517,11 @@ Twinkle.protect.callback.changeAction = function twinkleprotectCallbackChangeAct
 				list: Twinkle.protect.protectionTags,
 				event: Twinkle.protect.formevents.tagtype
 			});
+
+			var isTemplateNamespace = mw.config.get('wgNamespaceNumber') === 10;
+			var isWikipediaNamespace = mw.config.get('wgNamespaceNumber') === mw.config.get('wgNamespaceIds').project;
+			var isArticlesForDeletion = mw.config.get('wgTitle').indexOf('Articles for deletion/') === 0;
```

Comment:
I think we can combine these two in one line
```suggestion
			var isAFD = mw.config.get('wgPageName').indexOf('Wikipedia:Articles for deletion/') === 0;
```

Reply:
oops sorry, wgPageName encodes spaces as underscores. While we're at it, might as well use Morebits.pageNameNorm (which really does use spaces) and use String#startsWith()